Cargo.toml

```diff
@@ -174,7 +172,7 @@ lru = "0.7.2"
 memmap2 = "0.5"
 memoffset = "0.8"
 more-asserts = "0.2"
-near-account-id = { path = "core/account-id", features = ["internal_unstable"] }
+near-account-id = { git = "https://github.com/ruseinov/near-account-id", features = ["internal_unstable", "serde", "borsh"], branch = "ru/chore/len" }
```

Comment:
Strong no from me.
Imagine a nearcore developer making changes to the local version `near-account-id` only to realize that the code is using a branch in a remote git repo.

Reply:
This is a temporary fix, the idea is to point this at the main branch of near version of this, at least that is what we discussed with @frol . This was necessary as some of the pending PRs were not merged at the time.

Reply:
Thanks for the context, because I'm a simple engineer. When I see code for review, I review it :)

Reply:
> Imagine a nearcore developer working on a feature, and in the process making changes to the local version near-account-id. The developer will definitely be confused.

@nikurt The goal is to reach 1.0 release for near-account-id crate and use it across the whole ecosystem. There has been no development on this crate since it was decoupled into a self-sufficient crate within nearcore, and the goal for that decoupling in the first place was to use it outside of nearcore. It won't be pointing to git repo here, obviously, near-account-id will be used from crates.io. At this stage, we battle test the added AccountIdRef implementation, and once we confirm that everything works fine, we will release near-account-id 1.0, and then update this PR to use that, and request to review and merge it.

Reply:
Same here, no worries.